scenarios/kubernetes_e2e.py

```diff
@@ -594,6 +619,8 @@ def main(args):
 
     if args.aws:
         set_up_aws(mode.workspace, args, mode, cluster, runner_args)
+    elif args.kops:
```

Comment:
I'd like to get this to use a `--provider` flag, just like kubetest uses, but for now the logic is `--aws` => kops & aws, `--kops` => kops & gce.